libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -107,8 +107,6 @@ GCS_MAVLINK::init(AP_HAL::UARTDriver *port, mavlink_channel_t mav_chan)
 void
 GCS_MAVLINK::setup_uart(const AP_SerialManager& serial_manager, AP_SerialManager::SerialProtocol protocol, uint8_t instance)
```

Comment:
Any reason to not kill the argument and bubble the change the rest of the way up?

Reply:
Added benefit you don't have to pass the protocol into setup_uart realistically, which would let you remove the include for serial manager in GCS.h.

Reply:
I've added a commit to stop passing the serial manage around.

I've also removed the protocol parameter.

It was removing the include which sold me on it.

Reply:
> It was removing the include which sold me on it.

I cheated, I figured that would be the compelling argument for you :)